dist/fabric.js

```diff
@@ -29404,10 +29401,9 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
       this.styles = { };
       this.updateFromTextArea();
 
-      this.text = this.placeholder;
-      this.hiddenTextarea.value = this.placeholder;
+      this.text = this.placeholder || '';
```

Comment:
ideally we should not have `text: undefined` there, but if we do, everything will be crashing, so better to be safe